### DISCUSSION TITLE
Is there any gist/colab/repo showing micro pkg workflow?

### DISCUSSION THREAD

**Discussion by wmoreiraa** [kedro-org/kedro#2088] [Idea]
I've read the docs, but haven't found any examples showing how to use a "packaged pipeline" after pull. 
I think it would be a great addition to encourage this kind of workflow!

**Comment by stichbury** [comment]
Hi @wmoreiraa! Thanks for your question. Please can you raise it over on our Slack organisation (https://slack.kedro.org/) if you've not already done so. That's the place for questions about Kedro now and you'll get a response much more swiftly.
